go/consensus/tendermint/apps/scheduler/genesis.go

```diff
@@ -134,7 +126,7 @@ func (app *schedulerApplication) InitChain(ctx *abciAPI.Context, req types.Reque
 		ctx.Logger().Debug("adding validator to current validator set",
 			"id", id,
 		)
-		currentValidators = append(currentValidators, n.Consensus.ID)
+		currentValidators[n.Consensus.ID] = v.Power
```

Comment:
Shouldn't this instead derive voting power from stake and make sure it is correct?

Reply:
oh, in case something goes wrong in between?

Reply:
Yeah I know we derive this from our genesis doc anyway so it seems redundant, but on the other hand we also double-check that validators are in the registry.

Reply:
:vomiting_face: